sdk/cosmos/azure-cosmos-dotnet-benchmark/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,25 @@
+# This is the log4j configuration for benchmarks
+# Set root logger level to INFO and its default appender to be 'STDOUT'.
+rootLogger.level = info
+rootLogger.appenderRef.stdout.ref = STDOUT
+
+# Uncomment here and lines 21 - 25 to enable logging to a file as well.
+#rootLogger.appenderRef.logFile.ref = FILE
+
+property.logDirectory = $${sys:azure.cosmos.logger.directory}
+property.hostName = $${sys:azure.cosmos.hostname}
+
+logger.netty.name = io.netty
+logger.netty.level = off
+
+# STDOUT is a ConsoleAppender and uses PatternLayout.
+appender.console.name = STDOUT
+appender.console.type = Console
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d %5X{pid} [%t] %-5p %c - %m%n
+
+#appender.logfile.name = FILE
+#appender.logfile.type = File
+#appender.logfile.filename = ${logDirectory}/azure-cosmos-benchmark.log
```

Comment:
not important, just wonder should we use different file name here for .net?

Reply:
Fixed in next iteration